onebusaway-android/src/main/java/org/onebusaway/android/travelbehavior/constants/TravelBehaviorConstants.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2019 University of South Florida
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.onebusaway.android.travelbehavior.constants;
+
+import android.Manifest;
+
+public class TravelBehaviorConstants {
+
+    public static final String RECORD_ID = "tbRecordId";
+
+    public static final String USER_ID = "tbUserId";
+
+    public static final String USER_EMAIL = "tbUserEmail";
+
+    public static final String PARTICIPANT_SERVICE_RESULT = "STATUS OK";
+
+    public static final String REQUEST_CODE = "tbRequestCode";
+
+    public static final String TRIP_PLAN_COUNTER = "tripPlanCounter";
+
+    public static final String ARRIVAL_LIST_COUNTER = "arrivalCounter";
+
+    public static final String DEVICE_INFO_HASH = "deviceInfoHash";
+
+    public static final String DESTINATION_REMINDER_COUNTER = "destinationReminderCounter";
+
+    public static final String FIREBASE_ACTIVITY_TRANSITION_FOLDER = "activity-transitions/";
```

Comment:
`activity-transitions/` has a path character but the other folders don't - can we remove it for consistency?